src/components/dropdowns/dropdowns.html

```diff
@@ -160,17 +160,13 @@ <h3 class="docs-block__header">Example default usage (interactive)</h3>
                 </svg>
             </div>
             <div class="mint-dropdown__hole mint-dropdown__hole--inactive">
-                <div class="mint-label mint-label--emphasised mint-label--inactive mint-label--full-width">
-                    <div class="mint-label__text">
-                        Default label
-                    </div>
+                <div class="mint-text mint-text--gray-alt mint-text--obscure mint-text--xemphasised ">
```

Comment:
i would suggest using new practise here:
`<div style="display: none !improtant;">`
instead of 
`<div class="mint-text mint-text--gray-alt mint-text--obscure mint-text--xemphasised ">`